lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
OK, `mdfind make_manuals.out` shows three versions existing in `doc/ref`, `doc/tut` and `doc/changes`, looks like one each for the reference manual, tutorial and changes.

I've checked all three, and they look OK.  I can post the output if you want (about 50 lines in each).